contracts/AnchoredView/AnchoredView.sol

```diff
@@ -0,0 +1,254 @@
+pragma solidity ^0.6.6;
+pragma experimental ABIEncoderV2;
+
+import "./SymbolConfiguration.sol";
+import "../OpenOraclePriceData.sol";
+
+interface AnchorOracle {
+    function numBlocksPerPeriod() external view returns (uint); // approximately 1 hour: 60 seconds/minute * 60 minutes/hour * 1 block/15 seconds
+
+    function assetPrices(address asset) external view returns (uint);
+
+    struct Anchor {
+        // floor(block.number / numBlocksPerPeriod) + 1
+        uint period;
```

Comment:
do we only need period for checking the staleness of USDC? can we get rid of this struct just by storing a simple `lastUpdated` somewhere?

Reply:
this anchor struct is super specific to the version depending on the original oracle. It should not be used in the new view